### ISSUE TITLE
Incomplete/missing ALT text when using Ctrl+Enter to post instead of clicking/tapping

### ISSUE THREAD

**Issue by fasterthanlime** [cheeaun/phanpy#164]
**Describe the bug**
- Which site: phanpy.social

**To Reproduce**
1. Click on the "new post" (feather icon) button
2. Paste an image
3. Click into alt-text area, fill out some text
4. Hit ctrl-enter

**Expected behavior**
Post is posted with alt text

**Screenshots**

<img width="711" alt="image" src="https://github.com/cheeaun/phanpy/assets/7998310/a47b522a-424d-4907-ba92-10387c0373ca">

The alt text is not "updated" into the model until "onblur" in the textarea, I'm guessing. This is probably not a problem when clicking/tapping the "Post" button, but it is when using Ctrl-Enter.

**Desktop:**
 - OS: Windows
 - Browser: Edge
 - Version: 114.0.1823.43 (Official build) (64-bit) 

**Additional context**

I've noticed that in the past with incomplete alt text, and just recently with the "missing alt text" warning and it clicked in my head

**Comment by cheeaun** [comment]
@fasterthanlime Can you try wait for a second after stop typing, then hit ctrl-enter?

I can't seem to replicate this on my side. There is debounce of 500ms for the text fields, so you probably hit ctrl-enter within that split second 🤔

**Comment by fasterthanlime** [comment]
> There is debounce of 500ms for the text fields, so you probably hit ctrl-enter within that split second 🤔

That checks out - if I press Ctrl-Enter before the 500ms I get the warning / partial ALT text, but if I wait, then everything behaves as expected.

I'm curious what's the reasoning behind the debounce!

**Comment by cheeaun** [comment]
Okay, wow you typed _really_ fast 😮

The debounce was more technical, trying to fix an old performance issue and prevent React re-renders 😅. I've done a bunch of refactors since then, so I could probably revisit this, maybe either reduce the debounce or remove it 🙏

Thanks for reporting!

**Comment by fasterthanlime** [comment]
> Okay, wow you typed _really_ fast 😮

I am terminally online, yes (also writing is kind of my job). Glad there's a reasonable explanation and no big hurry on the fix!

**Comment by cheeaun** [comment]
@fasterthanlime I've (temporarily) reduced the debounce to 250ms, on production (probably need to reload, or check the menu to see if there's an update). Maybe can give it a whirl 🙈
